app/mailers/user_mailer.rb

```diff
@@ -8,4 +8,12 @@ def reset_password(user)
     @user = user
     mail to: user.email, subject: 'Reset password link'
   end
+
+  def answer_posted(question, question_user, answer, answer_user)
```

Comment:
avoid sending objects in mailer, pass id and load object there
and do we really need to pass everything?
answer_id should work here

Reply:
Load data in mailers using ID